### PR TITLE
Web Accessibity

### PR DESCRIPTION
**Pull request by Christabelle143**
# In this project my partner and I;
- Made our portfolio project accessible.
- The following points had no issues.
-  ### No issues found
- Color contrast
- Resize
- Interaction
- Moving content
- Multimedia
- The basic structure of the page

- ### Issues Found
- Page titles: https://github.com/Christabelle143/mobile-version/commit/4969c069b6f916ff2d48d1f7a842d04ab85d619f#
- Image text alternatives : https://github.com/Christabelle143/mobile-version/commit/02709867cc3ca4c3b55265266bfd3a24228ce08a# , https://github.com/Christabelle143/mobile-version/commit/02709867cc3ca4c3b55265266bfd3a24228ce08a# etc.
- Text headings: https://github.com/Christabelle143/mobile-version/commit/ab5fc30e640b2af3dc54349af68fecd4613cf587#

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,9 +5,9 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Mobile Version Skeleton</title>
-    <link rel="stylesheet" href="style.css">
-    <link rel="stylesheet" href="desk.css">
+    <title>Personal Portfolio</title>
```
Comment (prolajumokeoni) [inline index.html]: - [ ]  Kindly update title in the head to briefly describe the content of your page e.g; Christabelle's portfolio or Christabelle full-stack webdeveloper

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi Team,
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used 



## Required Changes ♻️

- [ ]  In your report   if issues were found for example page titles, Kindly add a link to the corresponding line of code in the Pull Request message

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Christabelle143) [reply index.html]: Hello @prolajumokeoni 
My partner and I added links to the corresponding line of codes which had issues and were modified.
Please you can check the above at the level of the issues found each point has a link to the issue in the codes

Comment (Christabelle143) [reply index.html]: Hey @prolajumokeoni 
Don't know if you meant another type of link

Comment (Christabelle143) [reply index.html]: hey... @prolajumokeoni

Comment (prolajumokeoni) [reply index.html]: Hi @Christabelle143   Yes, those are the type of comments I meant the type you have already in your pull request message, don't forget to add the page title's link to your  report 👍🏿

Comment (Christabelle143) [reply index.html]: I was able to change the page's title but how will  I change the link when I have already opened the pull request??

**Review by nicupop729 — APPROVED** [review]
Hi @Christabelle143 & @Elisha-Tetteyfio, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
 - [X] Correct GitHub flow practice :heavy_check_mark:
 - [X] Good commit messages :heavy_check_mark:
 - [X] Professional README :heavy_check_mark:
 - [X] Linters are passing :heavy_check_mark:
 - [X] Clean codebase :heavy_check_mark:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**